src/applications/personalization/view-dependents/tests/util/index.unit.spec.jsx

```diff
@@ -0,0 +1,36 @@
+import { expect } from 'chai';
+import { splitPersons, isServerError, isClientError } from '../../util/index';
+
+describe('View Dependents splitPersons Utility', () => {
+  const mockData = [
+    {
+      awardIndicator: 'N',
+    },
+    {
+      awardIndicator: 'N',
+    },
+  ];
+
+  it('should return an object with two arrays', () => {
+    const data = splitPersons(mockData);
```

Comment:
Another test we could do is to properly check for the presence of items in each array. Since both arrays are initialized in the `splitPersons` function, this test will always pass.

Reply:
That's very true. To test the data returned I added 2 expects for the data inside the returned array.